### PR TITLE
HCALDQM: Move channel quality lookup to dqmBeginLuminosityBlock (+increase pedestal RMS axis)

### PR DESCRIPTION
**Pull request by DryRun**
This PR accompanies https://github.com/cms-sw/cmssw/pull/25351, which implements LS-level granularity in HcalChannelQuality conditions. For HCAL DQM, this means that the HcalChannelQuality retrieval should happen in dqmBeginLuminosityBlock, rather than beginRun. 

I also include a small fix for the ADC axis on pedestal RMS plots, which was too small by the end of the year.

### CODE REVIEW

**Comment by abdoulline** [comment]
@smuzaffar 
looks like something went wrong with the PR processing initialization?

**Comment by smuzaffar** [comment]
cmsbot ping

**Comment by abdoulline** [comment]
@smuzaffar  Thanks

**Comment by andrius-k** [comment]
please test

**Comment by andrius-k** [comment]
+1

File: DQM/HcalCommon/src/DQClient.cc [hunk 1]
```diff
@@ -87,8 +87,16 @@ namespace hcaldqm
 			}
 		}
 
-		//	get the Channel Quality masks
+		// Initialize channel quality masks, but do not load (changed for 10_4_X, moving to LS granularity)
 		_xQuality.initialize(hashfunctions::fDChannel);
+	}
+	
+	/* virtual */ void DQClient::beginLuminosityBlock(DQMStore::IBooker&,
```
Comment (fabiocos) [inline DQM/HcalCommon/src/DQClient.cc]: @DryRun sorry why /* virtual */ ? If this is not needed please just remove it

File: DQM/HcalCommon/src/DQHarvester.cc [hunk 2]
```diff
@@ -90,8 +90,15 @@ namespace hcaldqm
 			}
 		}
 
-		//	get the Hcal Channels Quality for channels that are not 0
+		// Initialize channel quality masks, but do not load (changed for 10_4_X, moving to LS granularity)
 		_xQuality.initialize(hashfunctions::fDChannel);
+	}
+
+	/* virtual */ void DQHarvester::dqmBeginLuminosityBlock(
```
Comment (fabiocos) [inline DQM/HcalCommon/src/DQHarvester.cc]: @DryRun same here

File: DQM/HcalCommon/src/DQClient.cc (hunk 1 above)
Comment (DryRun) [reply DQM/HcalCommon/src/DQClient.cc]: Hi @fabiocos  - I have no idea, actually... this is from before my time. I'll remove it.

**Comment by andrius-k** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**